start-site/src/main/resources/application.yml

```diff
@@ -1020,6 +1020,21 @@ initializr:
             - rel: guide
               href: https://github.com/spring-cloud-samples/bookstore-service-broker
               description: Using Spring Cloud Open Service Broker
+        - name: App Broker
```

Comment:
While most Spring Cloud entries are unqualified, I find `App Broker` hard to resonate about, I don't know if `Cloud App Broker` would help with that.

Reply:
`Cloud App Broker` works for me if you feel like it makes it more clear. Spring Cloud App Broker is an opinionated implementation framework of Spring Cloud Open Service Broker (or the Open Service Broker API).